src/pybind/mgr/volumes/fs/operations/volume.py

```diff
@@ -227,6 +231,48 @@ def list_volumes(mgr):
         result.append({'name': f['mdsmap']['fs_name']})
     return result
 
+def info(mgr, path, fs_handle, volname):
+    """
+    Get all metadata of a volume.
+    """
+    vol_info_dict = {}
+    fs_map = mgr.get("fs_map")
+    for f in fs_map['filesystems']:
+        mdsmap = f['mdsmap']
+        if mdsmap['fs_name'] == volname:
+            gid = mdsmap['up']['mds_0']                    
+            vol_info_dict.update({'volname' : volname,
```

Comment:
Should we use `volume_name`?
(This will be shown on console output so we should have more user friendly words)

Reply:
Yeah,updated!!